src/cr/cube/cube.py

```diff
@@ -620,6 +638,30 @@ def _flat_values(self):  # pragma: no cover
 class _MeanMeasure(_BaseMeasure):
     """Statistical mean values from a cube-response."""
 
+    def __init__(self, cube_dict, all_dimensions, subvariables=None):
+        super(_MeanMeasure, self).__init__(cube_dict, all_dimensions)
+        self._subvariables = subvariables
+
+    @lazyproperty
+    def shape(self):
+        """tuple specifying measure shape.
+
+        If the provided variable (to calculate the means of) doesn't have subvariables,
+        this is the same as the parent class implementation.
+
+        If the variable (and consequently the measure data response) has subvariables,
+        then we need to alter the shape, by augmenting it with the number of subvars.
+        """
+        # Original shape: (m, n, . . .,)
+        shape = super(_MeanMeasure, self).shape
+
+        if self._subvariables is not None:
+            # There are subvariables in the provided variable => augment shape
+            # Shape now becomes: (m, n, . . ., n_subvars,)
+            shape += (len(self._subvariables),)
+
+        return shape
+
```

Comment:
I'd like to see a unit test for this specific property, maybe parametrised to cover the more meaningful edge cases

Reply:
Done